libs/components/src/lib/JobPostFormFirstPage/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { Form, Typography } from 'antd';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import {
+  StyledButton,
+  StyledInput,
+  StyledSelect,
+  StyledTextArea,
+} from '@freelance/components';
+import { yupResolver } from '@hookform/resolvers/yup';
+import useProperties from 'src/hooks/useProperties';
+
+import { InputsValues, schema } from '../constants';
+
+import { StyledErrorMessage } from './styles';
+
+export function JobPostFormFirstPage() {
+  const { categories, skills, getOptionsForSelectWithId } = useProperties();
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { Title, Text } = Typography;
```

Comment:
move out of component, you now initialise constant with each rerender

Reply:
yes, thanks